src/routes.js

```diff
@@ -8,13 +8,25 @@ import Home from 'views/Home';
 import Info from 'views/Info';
 import NotFound from 'views/NotFound';
 
+function trackLink() {
+  /* global Piwik */
+  try {
+    const tracker = Piwik.getTracker("http://analytics.hel.ninja/piwik/piwik.php", 2);
```

Comment:
K, what would be the best location for the tracking URL? All the way in kk.config and then get it to window via props?